apps/token-e2e/src/integration/flow/staking-flow.cy.js

```diff
@@ -278,11 +278,13 @@ context('Staking Tab - with eth and vega wallets connected', function () {
 
       cy.staking_validator_page_add_stake('1');
 
-      cy.get(vegaWalletNextEpochBalances, txTimeout).should(
-        'contain',
-        1.0,
-        txTimeout
-      );
+      // below check soemtimes fails since epoch duration is occasionally too quick for check to complete
+      // Looking into expanding epoch duration but for now to reduce flake: disabling this check
+      // cy.get(vegaWalletNextEpochBalances, txTimeout).should(
+      //   'contain',
+      //   1.0,
+      //   txTimeout
+      // );
```

Comment:
I would probably remove the code rather than commenting it out. But create a separate ticket to increase the epoch time and re-add the assertion when that's done.

Reply:
Done